rules/int_crawler.go

```diff
@@ -166,7 +166,8 @@ func (ic *intCrawler) singleRun(logger *zap.Logger) {
 }
 func (ic *intCrawler) processData(values map[string]string, logger *zap.Logger) {
 	api := &cacheReadAPI{values: values}
-	for k, v := range values {
+	for k := range values {
+		v := values[k]
```

Comment:
curious why this change?

Reply:
`rules/int_crawler.go:174:22: G601: Implicit memory aliasing in for loop. (gosec)`